js/index.js

```diff
@@ -79,6 +81,15 @@ const displayContent = ({ data }) => {
   //used destructuring for parsed.data => parsed object is destructured and {data} takes it's value.
   let id = 0;
```

Comment:
Won't all images that are appended from `appendLoop()` func have id = 0???

Reply:
Nope. The id is incremented in appendLoop().

Reply:
Omg, I didn't even notice. Mutating arguments is bad (very bad).
One of the main principles of functional programming and simply to avoid code smells is keeping your functions pure (do not mutate function arguments).
Pls refer here https://medium.com/javascript-scene/master-the-javascript-interview-what-is-a-pure-function-d1c076bec976#:~:text=Pure%20Functions,-A%20pure%20function&text=A%20dead%20giveaway%20that%20a,pure%20functions%2C%20that's%20a%20noop.&text=Pure%20functions%20take%20some%20input,output%20based%20on%20that%20input.

Reply:
fixed